src/Sulu/Component/PHPCR/PathCleanup.php

```diff
@@ -69,30 +81,30 @@ public function cleanup($dirty, $languageCode = null)
                 $dirty = str_replace($key, $value, $dirty);
             }
         }
-
-        $clean = strtolower($dirty);
-
-        // Inspired by ZOOLU
-        // delete problematic characters
-        $clean = str_replace('%2F', '/', urlencode(preg_replace('/([^A-za-z0-9\s\-_\/])/', '', $clean)));
-
         // replace multiple dash with one
-        $clean = preg_replace('/([-]+)/', '-', $clean);
+        $dirty = preg_replace('/([-]+)/', '-', $dirty);
 
         // remove dash before slash
-        $clean = preg_replace('/[-]+\//', '/', $clean);
+        $dirty = preg_replace('/[-]+\//', '/', $dirty);
 
         // remove dash after slash
-        $clean = preg_replace('/\/[-]+/', '/', $clean);
+        $dirty = preg_replace('/\/[-]+/', '/', $dirty);
 
         // delete dash at the beginning or end
-        $clean = preg_replace('/^([-])/', '', $clean);
-        $clean = preg_replace('/([-])$/', '', $clean);
+        $dirty = preg_replace('/^([-])/', '', $dirty);
+        $dirty = preg_replace('/([-])$/', '', $dirty);
+
+        $dirty = str_replace('//', '/', $dirty);
```

Comment:
can we use `preg_replace` here too? otherwise we will only replace two slashes with one but fail if there are three slashes 🙈

Reply:
should be something like `$dirty = preg_replace('/([\/]+)/', '-', $dirty);`